e2e/app/aou-elements/select-component.ts

```diff
@@ -28,19 +52,22 @@ export default class SelectComponent {
     return await innerText.jsonValue();
   }
 
-  // open Select dropdown with retries
-  private async open(retries: number): Promise<void> {
+  /**
+   * Open Select dropdown.
+   * @param {number} retries Retry count. Default is 1.
+   */
+  private async open(retries: number = 1): Promise<void> {
```

Comment:
I typically interpret retries=1 to mean you'll retry it once. Currently it seems to mean "retry 0 times".

Reply:
I'm not sure what to believe anymore,  looks like I implemented this a similar way to you :)

https://github.com/all-of-us/workbench/blob/81a1fc6b9b97d2dfe72c1d978621315bd71b1aea/ui/src/app/utils/retry.tsx#L8

How about `maxAttempts`?